domain-model-assistant/Assets/Components/Scripts/Diagram.cs

```diff
@@ -46,9 +46,9 @@ public class Diagram : MonoBehaviour
 
     Dictionary<string, GameObject> _namesToRects = new Dictionary<string, GameObject>();
 
-    Dictionary<string, List<Attribute>> idsToClassesAndAttributes = new Dictionary<string, List<Attribute>>();
+    Dictionary<string, List<Attribute>> classIdToAttributes = new Dictionary<string, List<Attribute>>();
 
-    Dictionary<string, string> idsToTypes = new Dictionary<string, string>();
+    Dictionary<string, string> AttrIdsToTypes = new Dictionary<string, string>(); //attribute ids to their types
```

Comment:
Keep capitalization consistent for instance vars (`AttrIdsToTypes` and `classIdToAttributes` don't match)